package-lock.json

```diff
@@ -1,11 +1,12 @@
 {
   "name": "contexture-react",
-  "version": "2.50.4",
+  "version": "2.50.5",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
-      "version": "2.50.4",
+      "name": "contexture-react",
```

Comment:
What version of npm are you using? You should be on 7.x.x

Reply:
After conversation with @rudolph9 we decided to leave it as is for now.